app/move/controllers/create/base.js

```diff
@@ -24,6 +27,39 @@ class CreateBaseController extends FormWizardController {
 
     next()
   }
+
+  async setMoveSummary(req, res, next) {
+    try {
+      const currentLocation = req.session.currentLocation
+      const sessionData = req.sessionModel.toJSON()
+      const person = req.sessionModel.get('person')
+      const { to_location: toLocationId } = sessionData
+
+      let toLocationDetail
+
+      if (toLocationId) {
+        toLocationDetail = await referenceDataService.getLocationById(
+          toLocationId
+        )
+      }
+
+      const moveSummary = presenters.moveToMetaListComponent({
+        ...sessionData,
+        from_location: currentLocation,
+        to_location: toLocationDetail,
+      })
+
+      res.locals = {
+        ...res.locals,
+        person,
+        moveSummary,
+      }
+
+      next()
+    } catch (err) {
```

Comment:
`error` instead of `err`? Everywhere else uses _error_ I believe and have started to favour this as it just reads a little nicer too.

Not a show-stopper though.

Reply:
I think the other benefit of `error` is that eslint will tell you off if we don't handle it correctly.